### DISCUSSION TITLE
Getting First Blockchain Developer Job - Advices

### DISCUSSION THREAD

**Discussion by Niferu** [Cyfrin/foundry-full-course-cu#503] [Q&A]
Hello everyone,

Let me introduce myself a bit first. I've spent past 8 years mastering finance industry by working for some of the top American banks in investment sector and trading various markets on my own. Since mid of 2020 I have started my programming journey from taking beginner Java course, after which I moved into programming trading bots and indicators for forex market (MQL5 language).

In early 2022 I have started my blockhain programmer journey and until now I have completed all 3 massive courses that Patrick prepared (Python, JavaScript and Foundry editions). I have also participated in 2 hackathons creating 3 pretty big projects. I'm also actively helping out others on those courses discussion tabs.

I'm applying to various positions and job offers since few months now, but no luck of getting even single interview. I think the problem is that I do not have any work records as even regular programmer and most of job offers are for senior+ (I didn't see any offers lower than senior in blockhain industry), so here comes my question:

Does any of you have some advices, sites, anything that could help people like me? I know it is time consuming process and I'm patient working on new projects, which I can attach to my CV/portfolio. I'm also trying fixing some open source projects issues etc. doing it every single day.

I will be very greateful for any advices/tips and tricks on how to get my first blockhain job. Below I'm linking sites, which I was using to search for offers:

https://wellfound.com/jobs
https://web3.career/
https://cryptojobslist.com/

Thank You In Advance!

**Comment by ChefAharoni** [comment]
Have you tried using LinkedIn?

**Reply by Niferu** [reply]
Yes, of course :) I'm getting some offers but mostly for Java, so I only partially match

**Comment by DukeLurkin** [comment]
Thanks for posting this.  

I've got even less experience having just completed a full-stack bootcamp a month ago and now finishing up the foundry course.  I do not see anywhere that there is entry level positions for anything in blockchain/defi.  All senior lvl+.  

I left the industry that I was in bc I wasn't doing what I wanted to be doing and now it seems like instead of coding for blockchains/defi like I want, the only opportunities are for JS, python, etc.. 

I wouldn't mind taking an entry lvl position coding something else but I feel like that would only set me back.  Maybe I'm wrong about that and if so I'm all ears!  

Anyway, I just wanted to add a comment so hopefully more ppl will see this post and to let you know you're not in that boat alone.  Some of us are even further back in the boat, lol.

**Reply by Niferu** [reply]
Thanks for that Sir! Any feedback in this field is much appreciated

**Comment by Niferu** [comment]
@alymurtazamemon, @cromewar, @PatrickAlphaC 

Sorry to call and bother you guys but I'd really appreciate your valuable thoughts on this matter as you are considered pros already working in this awesome industry. If you have any tricks and tips for programmers like me, I'd be incredibly grateful to hear them. Thanks in advance!

**Comment by alfheimrShiven** [comment]
My suggestion here would be to crack an opportunity as a full-stack developer first and get some professional experience there. Post which, attempt for blockchain based roles. See Web3, AI are niche technologies and the general trajectory to get into these fields will be after proving yourselves as a professional software developer first.

Disclaimer: I've not started applying for blockchain dev roles yet, so this suggestion is coming from my industry observation rather than experience.

Hope this helps @Neftyr @DukeLurkin 🤗

**Reply by Niferu** [reply]
Yup that is my plan also to first get regular programmer role if I fail finding blockchain one soon. Thanks for your thoughts!

**Reply by alfheimrShiven** [reply]
All the very best @Neftyr .

**Reply by DukeLurkin** [reply]
100% thanks for the input
